admin/admin-service/src/test/java/io/mosip/admin/controller/test/AdminControllerTest.java

```diff
@@ -113,7 +113,7 @@ public void setUp() throws Exception {
 	@Test
 	@WithUserDetails(value = "zonal-admin")
 	public void t002lostRidTest() throws Exception {
```

Comment:
Let's rename the test method name based on the agreed convention
methodNameUnderTest_scenarioName_ Result